playbook_arc_cluster.yml

```diff
@@ -1,47 +1,12 @@
 ---
 
-#########################################
-##    Some preps - PLAY 1
-#########################################
-
 - hosts: all
-  name: preps
-  gather_facts: yes
-  become: yes
-  tags: prepare
-
-  tasks:
-
-
-
-############################################
-##    ARC installation and related - PLAY 2
-############################################
-
-- hosts: frontend
   name: ARC installation
-  gather_facts: yes
   become: yes
   tags: 
     - installarc
     - arc
-
-  vars_files:
-    - roles/arc_frontend/vars/general.yml
-    - roles/arc_frontend/vars/arc_config.yml
-
-
-
-  tasks:
-
-   - name: Check dependent variables 
-     debug: 
-       msg: "You have variable site_type different than default: {{site_type}} but not set the runtimedir var. This must be set as site_type different than default will install files in the runtimedir."
-     failed_when: runtimedir|length == 0 and site_type != "default"
-
-
   roles:
-    - arc_frontend
-
-
-
+    - role: arc_frontend
+      sessionbasename: '/mnt/grid/sessiondir'
```

Comment:
Not convinced about putting these variables here - as there are many others the user needs to set too. What were your thoughts here?

Reply:
According to my tests in Alma 9 and Debian 11, to get a basic deployment with Slurm + ARC working with Infrastructure Manager I only need to set:
* `sessionbasename`
* `cachebasename`
* and for Debian exclusively: `lrms_bin_path`

This is similar to what we do in the deployment of HTCondor in Infrastructure Manager:
https://github.com/grycap/tosca/blob/main/artifacts/lrms/htcondor_fe_install.yml

How many other variables you think am I missing or will be need to be listed there?

Reply:
I see. 
At least we have to then auto-fill the lrms-type - if we say we just support slurm and htcondor first, this is a simple logic (well, the logic is simple anyway, but would require a long set of if-tests) 
Then I am assuming you are setting the minimal necessary variables to be changed to set up a default version of the ARC-CE. 
And then if the user wants more customisation he is expected to add more of the variables there in the variable list in the playbook - i.e. not touch the defaults/main.yml file? 
I guess that is fine, and is a bit more towards my wish for visibility (where I had the variables in the group_vars/all first. But your suggestion keeps it cleaner and easier integrateable with other playbooks and roles.

Reply:
> And then if the user wants more customisation he is expected to add more of the variables there in the variable list in the playbook - i.e. not touch the defaults/main.yml file?

That's exactly the purpose!

I have just added `slurm` to the examples.

If you agree, please merge so I can proceed with further refactoring.

Thanks!